GITFLOW.md

```diff
@@ -96,13 +96,21 @@ of Butterfree's python package to our private server.
 
 ### Update API Documentation
 
-If new information was added in the documentation in the release. You will need to update our hosted API Documentation. It's super simple, in the **documentation** branch  just bring the modifications from the master branch and run:
+If new information was added in the documentation in the release, maybe you will need to update our hosted Documentation. It's super simple, in the **docs** folder just apply the modifications and open a PR:
 
+If you want to test your changes locally, just run:
+ 
 ```bash
-make update-docs
+make test-docs
```

Comment:
I think that make docs would be a better name for the command.
Also, when this command is run it creates some files, right? In what path are they created and what's their pattern? Should we add this pattern to .gitignore to avoid them being commited?

Reply:
So, do you think that doesn't need the command [`make clean-docs`](https://github.com/quintoandar/butterfree/pull/201/files#diff-5d3c979bff19f72bf95f6d19e9065da0R110)? :thinking:

Reply:
just `make docs`